public/components/common/globalBreadcrumb/globalBreadcrumb.tsx

```diff
@@ -22,11 +22,17 @@ class WzGlobalBreadcrumb extends Component {
     $('#breadcrumbNoTitle').attr("title", "");
   }
   render() {
+    const options = [
+      { value: 'option_one', text: 'Option one' },
+      { value: 'option_two', text: 'Option two' },
+      { value: 'option_three', text: 'Option three' },
+    ];
+
+
     const container = document.getElementsByClassName('euiBreadcrumbs');
```

Comment:
Remove the `container` variable if not used because you removed the React portal.

Reply:
Removed!